pkg/flagset/flagset.go

```diff
@@ -267,6 +267,12 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"KONNECTD_INSECURE"},
 			Destination: &cfg.Konnectd.Insecure,
 		},
+		&cli.BoolFlag{
+			Name:        "no-tls",
```

Comment:
I'd call the flag simply `tls` and default it to true using `Value: true;` then it needs to be explicitly specified when you want to disable it